src/style.css

```
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.inputs {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  background-color: #fff;
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.container-bar {
+  position: relative;
+  margin-top: 20px;
+}
```

you could've applied this style to a `<label` element - than you wouldn't need any `<div` wrappers.
The minor difference is if you apply this style to `<label` - add display: block
```suggestion
.container-bar {
  display: block;
  position: relative;
  margin-top: 20px;
}
```